src/planscape/users/serializers.py

```diff
@@ -13,6 +15,27 @@ def custom_signup(self, request, user):
     user.last_name = self.validated_data.get('last_name', '')
     user.save(update_fields=['first_name', 'last_name'])
 
+class CustomPasswordResetSerializer(PasswordResetSerializer):
```

Comment:
Please add comments for these classes to describing their purpose for future maintainers.